s3torchconnectorclient/python/tst/integration/test_logging.py

```diff
@@ -0,0 +1,105 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+import pytest
+
+PYTHON_TEST_CODE = """
+import logging
+import os
+import sys
+
+os.environ["PYTHONUNBUFFERED"] = "1"
+os.environ["ENABLE_CRT_LOGS"] = "{0}"
+
+from s3torchconnector import S3MapDataset
+
+logging.basicConfig(
+    stream=sys.stdout,
+    format="%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s",
+)
+logging.getLogger().setLevel(logging.INFO)
+
+if __name__ == "__main__":
+    if len(sys.argv) != 3:
+        exit("The script needs an S3 uri and a region")
+    s3_uri = sys.argv[1]
+    region = sys.argv[2]
+    map_dataset = S3MapDataset.from_prefix(s3_uri, region=region)
+    obj = map_dataset[0]
+    assert obj is not None
+"""
+
+import subprocess
+
+
+@pytest.mark.parametrize(
+    "log_level, should_contain, should_not_contain",
+    [
+        (
+            "info",
+            ["INFO s3torchconnector.s3map_dataset"],
+            [
+                "DEBUG awscrt::AWSProfile",
+                "TRACE awscrt::AWSProfile",
+                "DEBUG awscrt::AuthCredentialsProvider",
+            ],
+        ),
+        (
+            "debug",
+            [
+                "INFO s3torchconnector.s3map_dataset",
+                "DEBUG awscrt::AWSProfile",
+                "DEBUG awscrt::AuthCredentialsProvider",
+            ],
+            ["TRACE awscrt::AWSProfile"],
+        ),
+        (
+            "trace",
+            [
+                "INFO s3torchconnector.s3map_dataset",
+                "DEBUG awscrt::AWSProfile",
+                "DEBUG awscrt::AuthCredentialsProvider",
+                "TRACE awscrt::event-loop",
+            ],
+            ["TRACE s3torchconnector.s3map_dataset"],
```

Comment:
I feel adding this would make us confused if we were to add trace level logs to S3MapDataset

Reply:
The logging level for Python is set to INFO in the test script. I'll add a clarifying comment.